packages/models/src/lib/category-config.unit.test.ts

```diff
@@ -176,7 +176,7 @@ describe('categoryConfigSchema', () => {
         ],
       } satisfies CategoryConfig),
     ).toThrow(
-      'In category informational, there has to be at least one ref with weight > 0. Affected refs: functional/immutable-data, lighthouse-experimental',
+      /In a category, there has to be at least one ref with weight > 0. Affected refs: \\"functional\/immutable-data\\", \\"lighthouse-experimental\\"/,
```

Comment:
Ah, I didn't realize Zod escapes quotation marks. In that case, unquoted ref slugs would be more readable after all, sorry for misdirecting you.

```suggestion
      /In a category, there has to be at least one ref with weight > 0. Affected refs: functional\/immutable-data, lighthouse-experimental/,
```

BTW since you're not using any special regex characters, you could just a plain string here (and elsewhere):

```suggestion
      'In a category, there has to be at least one ref with weight > 0. Affected refs: functional/immutable-data, lighthouse-experimental',
```